kie-pmml/src/main/java/org/kie/pmml/pmml_4_2/model/AbstractModel.java

```
@@ -96,11 +99,15 @@ public AbstractModel(String modelId, PMML4ModelType modelType, PMML4Unit owner,
     }
 
     protected void initFieldMaps() {
-        initMiningFieldMap();
-        initOutputFieldMap();
+    	try {
+            initMiningFieldMap();
+            initOutputFieldMap();
+    	} catch(Exception e) {
+    		logger.error(e.getMessage());
```

I am not sure if this is the right thing, because this just swallows the exception. Is there a mechanism how PMML propagates exception states to Drools engine? 

To clear the picture more - when an exception here is just logged, the method ends normally as nothing bad happened, so the program flow continues normally. This could have bad consequences, because a wrong state could be wrapped so it looks normal. From my point of view, when an exception happens somewhere here, it should be propagated to the method caller code layers in some form (either by some error state or as an exception). If PMML doesn't contain proper exception handling and propagation, maybe @tarilabs could provide you good tips how to handle this, because we have something similar in DMN engine.